src/foam/nanos/crunch/lite/ruler/CapableCreateApprovalsRuleAction.js

```diff
@@ -70,7 +70,7 @@ foam.CLASS({
       args: [
```

Comment:
This whole block could be just args: 'Context x, ApprovalRequest request, FObject obj, CapabilityJunctionPayload capablePayloadOjb'.

Reply:
Is it better for us to make it one line, since theres four arguments. Or is it easier to read as an array of maps

Reply:
I prefer the current as it's easier to inspect deltas on future commits. But I'll defer to Kevin.

Reply:
But it was just a suggestion / PSA, I don't care much either way. For new code I would choose one of the shorter formats as they're easier to read.